charts/k8ssandra/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: k8ssandra
 description: |
   Provisions and configures an instance of the entire K8ssandra stack. This includes Apache Cassandra, Stargate, Reaper, Medusa, Prometheus, and Grafana.
 type: application
-version: 1.0.1
+version: 1.0.1-1
```

Comment:
This should be 1.0.2

Reply:
You can use the script to help with these, ``scripts/helm-version.py --incr patch --chart k8ssandra``

Reply:
good to know.
 Fixed ✅